src/slack/shortcuts/replyAnonymously.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable camelcase */
+import { Middleware, SlackShortcutMiddlewareArgs, MessageShortcut } from '@slack/bolt';
+import logger from '../../logger';
+import { callbackIds } from '../constants';
+import { env } from '../../env';
+import { getPostAnonymousReplyBlock } from '../blocks/postAnonymousReply';
+import { app } from '../../app';
+
+export const replyAnonymously: Middleware<SlackShortcutMiddlewareArgs<MessageShortcut>> = async ({ shortcut, ack }) => {
+  ack();
+  try {
+    const originalPost = shortcut.message.text;
+    await app.client.views.open({
```

Comment:
I know the other pattern still uses this, but can you instead rely on the `client` object passed to this method so we don't need to use a `token` below?